### PR TITLE
Added the sample for Solace JMS OpenTelemetry Manual Instrumentation

### PR DESCRIPTION
**Pull request by mayur-solace**

### CODE REVIEW

**Comment by mayur-solace** [comment]
For review @Mrc0113

File: pom.xml [hunk 1]
```diff
@@ -6,11 +6,51 @@
     <version>1.0.1-SNAPSHOT</version>
     <name>Solace JMS 1.1 Examples</name>
     <description>Solace Messaging Examples for JMS 1.1</description>
+
+    <properties>
+        <opentelemetry.version>1.19.0</opentelemetry.version>
+        <opentelemetry-alpha.version>1.19.0-alpha</opentelemetry-alpha.version>
```
Comment (Mrc0113) [inline pom.xml:12]: what's the story with the alpha version? Is there another JIRA open to move to a non-alpha version in the future?

File: src/main/java/com/solace/samples/features/opentelemetry/manualinstrumentation/TopicPublisher.java [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+ * Sends a persistent message to a queue using Solace JMS API implementation.
+ * <p>
+ * Setup a Solace PubSub+ Broker and OpenTelemetry Collector as per tutorial  >
+ * https://codelabs.solace.dev/codelabs/dt-otel/index.html
+ * <p>
+ * This is the Publisher in the Publish-Subscribe messaging pattern.
+ */
+public class TopicPublisher {
+
+  final String TOPIC_NAME = "T/solace/tracing";
```
Comment (Mrc0113) [inline src/main/java/com/solace/samples/features/opentelemetry/manualinstrumentation/TopicPublisher.java]: please change topic to `solace/samples/jms/direct/pub/tracing`

File: src/main/java/com/solace/samples/features/opentelemetry/manualinstrumentation/TopicSubscriber.java [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+ * Subscribes to message published to a topic using Solace JMS 1.1 API implementation.
+ * <p>
+ * Setup a Solace PubSub+ Broker and OpenTelemetry Collector as per tutorial  >
+ * https://codelabs.solace.dev/codelabs/dt-otel/index.html
+ * <p>
+ * This is the Subscriber in the Publish-Subscribe messaging pattern.
+ */
+public class TopicSubscriber {
+
+  private static final String TOPIC_NAME = "T/solace/tracing";
```
Comment (Mrc0113) [inline src/main/java/com/solace/samples/features/opentelemetry/manualinstrumentation/TopicSubscriber.java]: solace/samples/jms/direct/pub/tracing

File: pom.xml (hunk 1 above)
Comment (mayur-solace) [reply pom.xml:12]: OpenTelemetry releases `opentelemetry-semconv` and few other as alpha only, as they may be subject to change.

File: src/main/java/com/solace/samples/features/opentelemetry/manualinstrumentation/TopicSyncReceiver.java [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+ * Subscribes to message published to a topic using Solace JMS 1.1 API implementation.
+ * <p>
+ * Setup a Solace PubSub+ Broker and OpenTelemetry Collector as per tutorial  >
+ * https://codelabs.solace.dev/codelabs/dt-otel/index.html
+ * <p>
+ * This is the Subscriber in the Publish-Subscribe messaging pattern.
+ */
+public class TopicSyncReceiver {
+
+  private static final String TOPIC_NAME = "T/solace/tracing";
```
Comment (Mrc0113) [inline src/main/java/com/solace/samples/features/opentelemetry/manualinstrumentation/TopicSyncReceiver.java]: solace/samples/jms/direct/pub/tracing

File: src/main/java/com/solace/samples/jms/snippets/HowToImplementManualInstrumentation.java [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+import java.util.function.Consumer;
+import javax.jms.Message;
+import javax.jms.MessageProducer;
+import javax.jms.Topic;
+
+/**
+ * Example how to manually instrument Solace JMS publisher and receiver. All examples assume usage
+ * of a newer Solace JMS API v10.18.0 or newer.
+ */
+public class HowToImplementManualInstrumentation {
```
Comment (Mrc0113) [inline src/main/java/com/solace/samples/jms/snippets/HowToImplementManualInstrumentation.java]: Please rename to `HowToImplementTracingManualInstrumentation` to be specific that this is about tracing like we did for JCSMP as well.

**Review by Mrc0113 — CHANGES_REQUESTED** [review]
Overall this looks good! Just minor name/topic changes.

**Comment by Mrc0113** [comment]
Thanks @mayur-solace. are you still making other changes or good to go now?

**Comment by mayur-solace** [comment]
> Thanks @mayur-solace. are you still making other changes or good to go now?

good to merge. I'm not making anymore changes.

**Pull request merged**